aiosparql/client.py

```diff
@@ -178,6 +185,37 @@ def _pretty_print_query(self, query: str) -> str:
                     explanation=explanation)
             return await resp.json()
 
+    def _crud_request(self, method, graph=None, data=None):
```

Comment:
HTTP body.

The graph store protocol is basically a crud endpoint where you post/delete/get raw data in any format you want (RDF XML, Turtle, etc...). 

Just realized now that there is no parameter to set the format. Maybe I should start adding integration tests.